po/zh_CN.po

```diff
@@ -331,11 +331,11 @@ msgstr "默认解释器"
 
 #: ../data/prefs.glade.h:49
 msgid "_Run command as a login shell"
-msgstr "运行命令作为登录外壳(_R)"
```

Comment:
Maybe, but I try to press "R" and nothing happen. Then I change my system language to English, 
 and the text there just `"Reset Compatibibility***"`, the R is just a normal text, and no tailing (_R), so I delete the (_R) in the translation.